lib/developer_portal/lib/liquid/tags/essential_assets.rb

```diff
@@ -1,18 +1,13 @@
+# frozen_string_literal: true
+
 module Liquid
   module Tags
     class EssentialAssets < Liquid::Tags::Base
-      include Liquid::Tags::ContentFor::ContentFor
-      nodoc!
-
-      # this makes versioning harder, delete it
+      deprecated 'This is outdated and should not be use anymore.'
 
-      def render(context)
-        [
-            stylesheets.render(context),
-            render_erb(context, "shared/essential_assets"),
-            javascripts.render(context)
-        ].join.html_safe
-      end
+      # This exists for backwards compatibility. Some very old dev portals may still be using tag
+      # {% essential_assets %} and thus loading old libraries and overriding our updates.
+      def render(context); end
```

Comment:
Those old sites potentially calling the tag would require the assets as well, right? Wouldn't this be a breaking change anyway?

Reply:
I understand that those assets are "required" by this `render` method, so if we keep it empty, the assets will not be loaded.

By the way the above Liquid `container` was also deprecated in 2012 - https://github.com/3scale/system/commit/c5fcf3e72b4c340a716f49d92692b31f593645a7

I think it's pretty safe to get rid of this.

Reply:
> I understand that those assets are "required" by this render method, so if we keep it empty, the assets will not be loaded.

Yes, but if the site was calling `essential_assets` it because it expected to load some JS code included in that tag, right? so not including it would break the site.

Reply:
Ah, well... you mean that the code would rely on the JS functionality in those assets?.

I don't think we need to care about it, as I really doubt there are actual portals that use those liquids.

Reply:
> Ah, well... you mean that the code would rely on the JS functionality in those assets?.
> 
> I don't think we need to care about it, as I really doubt there are actual portals that use those liquids.

This was indeed a breaking change for old saas portals :upside_down_face:

Reply:
> This was indeed a breaking change for old saas portals 🙃

Oops... I guess I need to stop making bold assumptions :upside_down_face: